mern/client/src/Components/TranscriptScroller/transcript-scroller.component.js

```diff
@@ -2,19 +2,15 @@ import React, { useContext } from "react";
 import GenericButton from "../Buttons/GenericButton";
 import styles from "./transcript-scroller.component.module.css";
 import { ScrollerContext } from "../../Contexts/scrollerProvider";
-import GetFlowData from "../../utils/export";
+import exportController from "../../utils/Controller/exportController";
 import { QuestionContext } from "../../Contexts/questionProvider";
 
 export default function Scrollbar() {
   const [, , , , allQuestions, , , ,] = useContext(QuestionContext);
-
   const downloadFile = () => {
     // Gets the flow data as a list of JSON object of QA pairs
-
```

Comment:
Make sure to search through the codeBase for all unnecessary console.logs and ensure they are removed before submission.